### ISSUE TITLE
Phoenix Web wallet not updating Transactions

### ISSUE THREAD

**Issue by pir8radio** [signum-network/signum-node#512]
The javascript is firing on cue and getting the json responses, however the responses are not getting painted in the transaction screen.       I only have one js error shown in the screenshot.   May be related, I have not dug into it.    But looking at the screenshot in chrome dev mode, you can see the page on the left shows stale info, while the XHR data came back and is updated, it is just not making it to the web interface..    A refresh of the page shows the correct info, but it is not auto refreshing.  

![image](https://user-images.githubusercontent.com/16748696/123570780-8f092080-d78e-11eb-8f26-5acc7e8d65a6.png)

**Comment by th1nkNXT** [comment]
I have the same problem. Due to a JS error the automatic refresh of the page doesn't work (Raspbian linux, Chromium). I will take a look into it. Maybe i can find the cause of that error.

**Comment by th1nkNXT** [comment]
I had another problem and i created an issue in the repo for phoenix:
https://github.com/signum-network/phoenix/issues/1531

TL;DR: It was a problem with the bundled version of the phoenix wallet which was include in the signum-node 3.1.1. Try to use the latest version of the phoenix wallet (>=1.2.2) and check if your problem is gone.

**Comment by th1nkNXT** [comment]
@pir8radio 
Hi. Did you tested it with phoenix wallet (>=1.2.2)? Can this issue be closed then? Thanks for your feedback.

**Comment by pir8radio** [comment]
> @pir8radio
> Hi. Did you tested it with phoenix wallet (>=1.2.2)? Can this issue be closed then? Thanks for your feedback.

This issue is for the bundled "WEB" wallet..    has this issue been corrected in the next BRS release?    I do not use the stand alone wallet..  only web.

**Comment by th1nkNXT** [comment]
Yes, there were errors in the bundled version of the Phoenix Wallet (Signum Node v3.1.1 uses Phoenix version 1.2.1). 
You can install the newer version into a **separate** folder, see https://github.com/signum-network/phoenix/releases.

Please be aware that you cannot just copy the release of the Phoenix Wallet into the same folder structure of your Signum Node installation as there have modifications to be made to the files (otherwise it will not work). That is the reason why you should install it separately.

I think a new bugfixed version will be released with a new Signum Node release but i do not know any dates here.

**Comment by ohager** [comment]
@pir8radio can we close this?

**Comment by pir8radio** [comment]
Yes it looks like XHR requests are updating the transactions, now working!   thanks!!
